README.rst

```diff
@@ -0,0 +1,128 @@
+.. use rst_include to compile the README.rst on GitHub.
+   pip install rst-include
+   rst_include include -s README_src.rst -t README.rst
+
+.. |pypi| image:: https://badge.fury.io/py/benzina.svg
+   :scale: 100%
+   :target: https://pypi.python.org/pypi/benzina
+
+.. |docs| image:: https://readthedocs.org/projects/docs/badge/?version=latest
+   :scale: 100%
+   :target: https://benzina.readthedocs.io/en/latest
+
+|pypi| |docs|
+
+=================
+Бензина / Benzina
+=================
+
+Description of the project
+==========================
+
+Benzina is an image loading library that accelerates image loading and preprocessing
```

Comment:
@satyaog @fosterrath-mila I'm about to squash and merge everything, but would like to know why is there seeming copy-paste between this file and the contents of `doc/source/*`. For instance, these paragraphs and tables.